pxtlib/util.ts

```diff
@@ -737,7 +737,7 @@ namespace ts.pxtc.Util {
 
     }
 
-    function normalizeLanguageCode(code: string): string {
+    export function normalizeLanguageCode(code: string): string {
         if (!/^(es|pt|si|sv|zh)/i.test(code))
```

Comment:
we should revisit this...

Reply:
Agreed that it's weird... Changed the normalization logic to instead ensure casing is consistent: `xx` and `xx-XX`